4-Classification/2-Classifiers-1/translations/README.es.md

```diff
@@ -0,0 +1,245 @@
+
+- Modelos lineales
+- Máquinas de vectores de soporte
+- Descenso de gradiente estocástico
+- Vecinos más cercanos
+- Procesos Gaussianos
+- Árboles de decisión
+- Métodos de conjunto (clasificador de votos)
+- Algoritmos multiclase y multisalida (clasificación multiclase y multietiqueta, clasificación multiclase-multisalida)
+
+> También puedes usar [redes neuronales para clasificar los datos](https://scikit-learn.org/stable/modules/neural_networks_supervised.html#classification), pero eso está fuera del alcance de esta lección.
+
+### ¿Qué clasificador usar?
+
+Así que, ¿qué clasificador deberías elegir? A menudo, el ejecutar varios y buscar un buen resultado es una forma de probar. Scikit-lean ofrece una [comparación lado a lado](https://scikit-learn.org/stable/auto_examples/classification/plot_classifier_comparison.html) en un conjunto de datos creado, comparando KNeighbors, SVC two ways, GaussianProcessClassifier, DecisionTreeClassifier, RandomForestClassifier, MLPClassifier, AdaBoostClassifier, GaussianNB y QuadraticDiscrinationAnalysis, mostrando los resultados visualizados:
+
+![Comparación de clasificadores](../images/comparison.png)
+> Gráficos generados en la documentación de Scikit-learn
+
+> AutoML resuelve este problema de forma pulcra al ejecutar estas comparaciones en la nube, permitiéndote elegir el mejor algoritmo para tus datos. Pruébalo [aquí](https://docs.microsoft.com/learn/modules/automate-model-selection-with-azure-automl/?WT.mc_id=academic-15963-cxa)
+
+### Un mejor enfoque
+
+Una mejor forma a estar adivinando, es seguir las ideas de esta [hoja de trucos de ML](https://docs.microsoft.com/azure/machine-learning/algorithm-cheat-sheet?WT.mc_id=academic-15963-cxa). Aquí, descubrimos que, para nuestro problema multiclase, tenemos algunas opciones:
+
+![Hoja de trucos para problemas multiclase](../images/cheatsheet.png)
+> Una sección de la hoja de trucos de algoritmos de Microsoft, detallando opciones de clasificación multiclase.
+
+✅ !Descarga esta hoja de trucos, imprímela y cuélgala en tu pared!
+
+### Razonamiento
+
+Veamos si podemos razonar nuestro camino a través de diferentes enfoques dadas las restricciones que tenemos:
+
+- **Las redes neuronales son muy pesadas**. Dado nuestro conjunto de datos limpio aunque mínimo, y el hecho que estamos ejecutando el entrenamiento de forma local vía los notebooks, las redes neuronales son demasiado pesadas para esta tarea.
+- **Sin clasificador de dos clases**. No usamos clasificador de dos clases, por lo que descarta un uno-contra-todos.
+- **El árbol de decisión o la regresión logística podría funcionar**. Un árbolo de decisión podría funcionar, o la regresión logística para datos multiclase.
```

Comment:
Small typo, should be "Un árbol de decisión podría funcionar, o la regresión logística para datos multiclase."

Reply:
Thanks for your help!